LICENSE

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
This change (and some others in this PR) affect the text of a license file. This seems unlikely to have been the intent. For example, the link here and below are supposed to point to the canonical version of this very license, and they do, and that canonical version does not use "https".

This causes downstream problems, for example, it means that Flutter's license script considers Abseil's Apache license to be non-canonical, causing us to include two copies of the Apache license in our license screen, one for Abseil files that refer to the canonical license directly in their header (and for everything else in Flutter that's covered by the Apache license), and one for Abseil files that don't have an explicit license statement and therefore end up covered by this file which differs from the canonical version by two bytes (the two "s" characters added by this PR).

Reply:
We heard from numerous internal Google teams about the same issue. In hindsight, this change was not worth it, and it is great example of [Hyrum's Law](https://www.hyrumslaw.com/) in action.

At this point, this change is over two years old. We won't be changing it back, but we will use it as an example of why we need to avoid unnecessary churn in the future.

Reply:
Thanks for the acknowledgement.

Reply:
This is great feedback -- surprised I decided to do this back then but definitely useful information to know!